lib/src/Uncertainty/Algorithm/MetaModel/Kriging/KrigingEvaluation.cxx

```diff
@@ -201,15 +247,56 @@ struct KrigingEvaluationSampleFunctor
   } // operator()
 }; // struct KrigingEvaluationSampleFunctor
 
+struct KrigingEvaluationSampleFunctor1D
+{
+  const Sample & input_;
+  Sample & output_;
+  const KrigingEvaluation & evaluation_;
+  UnsignedInteger trainingSize_;
+
+  KrigingEvaluationSampleFunctor1D(const Sample & input,
+                                   Sample & output,
+                                   const KrigingEvaluation & evaluation)
+    : input_(input)
+    , output_(output)
+    , evaluation_(evaluation)
+    , trainingSize_(evaluation.inputSample_.getSize())
+  {
+    // Nothing to do
+  }
+
+  inline void operator()( const TBB::BlockedRange<UnsignedInteger> & r ) const
+  {
+    const UnsignedInteger inputDimension = input_.getDimension();
+    const UnsignedInteger start = r.begin();
+    const UnsignedInteger size = r.end() - start;
+    Scalar value = 0.0;
+    for (UnsignedInteger i = 0; i != size; ++i)
+    {
+      for (UnsignedInteger j = 0; j < trainingSize_; ++j)
+        value += evaluation_.covarianceModel_.getImplementation()->computeAsScalar(input_.getImplementation()->data_begin() + (start + i) * inputDimension, evaluation_.inputSample_.getImplementation()->data_begin() + (j * inputDimension) ) * evaluation_.gamma_(j, 0);
```

Comment:
Sorry @regislebrun I introduced an issue 
Could you fix it please ?
```
-    Scalar value = 0.0;
     for (UnsignedInteger i = 0; i != size; ++i)
     {
       for (UnsignedInteger j = 0; j < trainingSize_; ++j)
-        value += evaluation_.covarianceModel_.getImplementation()->computeAsScalar(input_.getImplementation()->data_begin() + (start + i) * inputDimension, evaluation_.inputSample_.getImplementation()
-      output_(start + i, 0) = value;
+        output_(start + i, 0) += evaluation_.covarianceModel_.getImplementation()->computeAsScalar(input_.getImplementation()->data_begin() + (start + i) * inputDimension, evaluation_.inputSample_.get
```

Reply:
done